### PR TITLE
Actions: Deprecate old action dispatch and lint file

### PR DESCRIPTION
**Pull request by badosu**

### CODE REVIEW

File: luaui/actions.lua [hunk 1]
```diff
@@ -262,7 +243,7 @@ function actionHandler:TextAction(line)
   end
   -- remove the command from the words list and the raw line
   table.remove(words, 1)
-  local _,_,line = string.find(line, "[^%s]+[%s]+(.*)")
+  _,_,line = string.find(line, "[^%s]+[%s]+(.*)")
```
Comment (Beherith) [inline luaui/actions.lua:246]: Why is line global here?

**Review by Beherith — CHANGES_REQUESTED** [review]
I'm just wondering, does this deprecate anything that could concievably be in common use?

File: luaui/actions.lua (hunk 1 above)
Comment (sprunk) [reply luaui/actions.lua:246]: It isn't, it's the local var from line 238.

Comment (badosu) [reply luaui/actions.lua:246]: _ is reserved keyword

We are fixing a redundant `local` variable assignment

Comment (Beherith) [reply luaui/actions.lua:246]: Thanks for clearing that up!

**Pull request merged**